instrumentation/logback/logback-appender-1.0/javaagent/src/test/java/io/opentelemetry/instrumentation/logback/appender/v1_0/LogbackTest.java

```diff
@@ -219,6 +222,25 @@ void testMdc() {
                     .containsEntry(SemanticAttributes.CODE_FILEPATH, "LogbackTest.java"));
   }
 
+  @Test
+  public void testMarker() {
+
+    String markerName = "aMarker";
+    Marker marker = MarkerFactory.getMarker(markerName);
+
+    ghiLogger.info(marker, "Message");
```

Comment:
ok to reuse `abcLogger` here, similar to mdc test above

Reply:
Done